fribidi-ignore-warnings.patch

```diff
@@ -0,0 +1,12 @@
+diff -up fribidi/configure.ac.autoxxx fribidi/configure.ac
+--- fribidi/configure.ac.autoxxx	2017-11-26 20:08:15.729211631 +0000
++++ fribidi/configure.ac	2017-11-26 20:30:00.418422977 +0000
+@@ -122,7 +122,7 @@ AC_CHECK_SIZEOF(wchar_t)
+ # Checks for compiler characteristics.
+ changequote(,)dnl
+ if test "x$GCC" = "xyes"; then
+-  CFLAGS="$CFLAGS -Wall -ansi "
++  CFLAGS="$CFLAGS -ansi -Wno-deprecated-declarations"
```

Comment:
The better solution to this is finding the `-Werror` and removing it.

Reply:
With `-Werror` it fails like this https://pastebin.ubuntu.com/26053660/

It is therefore acceptable to have the CFLAGS as `CFLAGS="$CFLAGS -Werror -Wno-deprecated-declarations -ansi "` ? (As this then works)

Reply:
In other words, the problem is that there is -Werror defined somewhere and it shouldn't be except for development purposes. It should be removed instead of adding warious `-Wno-` flags.

Reply:
Ah! I think the `-Wno-deprecated-declarations` isn't actually required, I just had it there when trying to use `-Werror` as well. OK I'll remove that `-Wno-deprecated-declarations` so the cflags can become `CFLAGS="$CFLAGS -ansi "`. (I hope this is what you mean, and I haven't got confused).